### ISSUE TITLE
Where are the Perl 6 communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/perl6#289]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Perl 6 supported by one or more large organizations?
* Does Perl 6 have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Perl 6? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Perl 6? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Perl 6? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Perl 6?
* Are there popular newsletters for Perl 6?
* Is Perl 6 taught at programming bootcamps? (If so, what are some examples?)
* Is Perl 6 taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Perl 6?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by h-m-m** [comment]
In re popular conferences, I'm familiar with [The Perl Conference](https://perlconference.us) 
The conference has been co-organized by local Perlmonger groups, more information about those user groups at https://www.pm.org/
Perlmonger groups are under the umbrella of The Perl Foundation http://www.perlfoundation.org/

I don't know Perl well enough anymore to be a mentor, and I stopped before Perl 6. That said, I talk regularly with a couple people enthusiastic about Perl 6, and they are my source for this info.

**Comment by kytrinyx** [comment]
Thanks @h-m-m!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼

**Comment by mienaikage** [comment]
* Are there popular newsletters for Perl 6?
https://p6weekly.wordpress.com/

* Do you know of specific communities (online or offline) that are enthusiastic about Perl 6? (Chat communities, forums, meetups, student clubs, etc)
https://www.reddit.com/r/perl6/
The perl6 IRC and mailing list found here: https://perl6.org/community/
